ide/projectui/src/org/netbeans/modules/project/ui/OpenProjectList.java

```diff
@@ -1379,6 +1376,7 @@ private boolean doOpenProject(final @NonNull Project p) {
             public @Override Boolean run() {
             log(Level.FINER, "already opened: {0} ", openProjects);
             for (Project existing : openProjects) {
+                // TODO double tap ???
```

Comment:
Proposed improved comment:
`// An old hack due to broken equals() contract; see https://bz.apache.org/netbeans/show_bug.cgi?id=156536`